docs/queries/function_filter.rst

```diff
@@ -0,0 +1,49 @@
+Function Filter Query
+==========================
+
+A query that retrieves all documents with a positive score calculated by the script.
+
+Proto definition:
+
+.. code-block::
+
+   message FunctionFilterQuery {
+       Script script = 1; // script definition to compute a custom document score
+   }
+
+   message Script {
+       string lang = 1; // script language
```

Comment:
If I'm not mistaken core nrtsearch doesn't support any scripting language out of the box and these languages can be supported using custom plugins. Maybe we can give some examples such as `painless`, `groovy` and mention that they should be enabled separately.

Reply:
Nrtsearch does support Javascript expressions https://github.com/Yelp/nrtsearch/blob/master/src/main/java/com/yelp/nrtsearch/server/luceneserver/script/js/JsScriptEngine.java
Will make it clearer as we improve the documentation